ios/PolkadotVault/Backend/CreateDerivedKeyService.swift

```diff
@@ -28,6 +47,44 @@ final class CreateDerivedKeyService {
         self.navigation = navigation
     }
 
+    func createDerivedKeys(
+        _ seedName: String,
+        networks: [MmNetwork],
+        completion: @escaping (Result<Void, CreateDerivedKeyError>) -> Void
+    ) {
+        let pathAndNetworks: [(path: String, network: MmNetwork)] = networks
+            .map { (path: "//\($0.title)", network: $0) }
```

Comment:
Please update, when https://github.com/paritytech/parity-signer/pull/1880 is merged

Reply:
Updated to:
```swift
            .map { (path: $0.pathId, network: $0) }
```